Compilation error with gcc 13.3.0

## Steps to reproduce

```bash
git clone https://github.com/robotics-upo/lightsfm.git
cd lightsfm
make
g++ -c -Wall -O3  -std=gnu++11 -Iinclude  src/test.cpp -o obj/test.o
In file included from src/test.cpp:28:
include/sfm.hpp:101:8: error: ‘list’ in namespace ‘std’ does not name a template type
  101 |   std::list<Goal> goals;
      |        ^~~~
include/sfm.hpp:31:1: note: ‘std::list’ is defined in header ‘<list>’; did you forget to ‘#include <list>’?
   30 | #include <vector>
  +++ |+#include <list>
   31 | 
```

## Environment

- Ubuntu 24.04
- gcc 13.3.0

--- a/include/sfm.hpp
+++ b/include/sfm.hpp
@@ -28,6 +28,7 @@
 #include <cmath>
 #include <unordered_map>
 #include <vector>
+#include <list>
 
 namespace sfm {
 struct Forces {

